README.md

```diff
@@ -1181,4 +1181,4 @@ See [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Copyright and License
 
-Copyright (c) 2012-2014 Tim Vandecasteele and contributors. See [LICENSE.txt](LICENSE.txt) for details.
+Copyright (c) 2012-2016 Tim Vandecasteele and contributors. See [LICENSE.txt](LICENSE.txt) for details.
```

Comment:
Tim isn't stripped from it, he is included in contributors (without any contributions), and he is a member of ruby-grape, so it should be enough → make a diff against 1.2 branch and a blame → most of the code has significant changes